src/commands/force/source/convert.ts

```diff
@@ -93,13 +94,29 @@ export class Convert extends SourceCommand {
       },
     });
 
+    const packageName = this.getFlag<string>('packagename');
+    const outputDirectory =
+      this.getFlag<string>('outputdir') === './' ? `metadataPackage_${Date.now()}` : this.getFlag<string>('outputdir');
```

Comment:
Maybe we should resolve the outputdir.  A value of `'.'` here throws an error.  We might also want to ensure any directory specified here is empty before continuing or an error will be thrown during rename.